client/src/components/navLink/NavLink.jsx

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import classes from './navLink.module.css';
+import PropTypes from 'prop-types';
 
 function NavLink(props) {
+  console.log(typeof props.theme);
   return (
-    <Link className={`${classes.link} ${props.theme}`} to={props.path}>
+    <Link className={classes.link} to={props.path}>
       {props.text}
```

Comment:
Do not limit yourself to just prop "text" - what of you want to pass span element with styles applied? You can always use children